features/step_definitions/sitewide_settings_steps.rb

```diff
@@ -16,20 +16,20 @@
   visit "/government/how-government-works"
 end
 
-Then(/^I should (not )?see the minister counts$/) do |negate|
-  if negate
-    expect(page).to_not have_selector(".feature-ministers")
-  else
-    expect(page).to_not have_selector(".feature-ministers")
-  end
+Then(/^I should see the minister counts$/) do
+  expect(page).to have_selector(".feature-ministers")
 end
 
-Then(/^I should (not )?see a reshuffle warning message$/) do |negate|
-  if negate
-    expect(page).to_not have_content("Test minister <a rel=\"external\" href=\"http://example.com\">reshuffle</a> message")
-  else
-    expect(page).to have_content("Test minister reshuffle message")
-  end
+Then(/^I should not see the minister counts$/) do
+  expect(page).to_not have_selector(".feature-ministers")
+end
+
+Then(/^I should see a reshuffle warning message$/) do
+  expect(page).to have_content("Test minister reshuffle message")
+end
+
+Then(/^I should not see a reshuffle warning message$/) do
+  expect(page).to_not have_content("Test minister <a rel=\"external\" href=\"http://example.com\">reshuffle</a> message")
```

Comment:
I'm not sure this assertion is correct as the affirmative situation above doesn't contain the HTML for the link.  Also, I'm pretty sure `have_content` [only matches the text, not the HTML](https://www.rubydoc.info/gems/capybara/Capybara%2FRSpecMatchers:have_content).

Reply:
You're correct, good catch! `have_content` only matches text, not HTML and so the first statement passes even though it does contain a link in the HTML. The second step was passing but not checking the right content, due to looking for raw HTML. Have updated this.